README.md

```diff
@@ -1,5 +1,13 @@
-# Fragments-Download
-This script downloads anyurl by fragmenting the file and increases speed drastically.
+# All purpose downloader 
+## Powered by CRUx - The computing and programming club 
 
-usage: python3 frag.py URL
+### BITS Pilani Hyderabad Campus
```

Comment:
I think college name is kind of essential because the club is because of the college. Just an opinion